x-pack/legacy/plugins/siem/public/components/ml_popover/api.tsx

```diff
@@ -176,63 +219,24 @@ export const stopDatafeeds = async (
 };
 
 /**
- * Fetches Job Details for given jobIds
+ * Fetches a summary of all ML jobs currently installed
  *
- * @param jobIds
- * @param headers
+ * @param signal to cancel request
  */
-export const jobsSummary = async (
-  jobIds: string[],
-  headers: Record<string, string | undefined>,
-  signal: AbortSignal
-): Promise<Job[]> => {
+export const getJobsSummary = async (signal: AbortSignal): Promise<JobSummary[]> => {
   const [kbnVersion] = useKibanaUiSetting(DEFAULT_KBN_VERSION);
   const response = await fetch(`${chrome.getBasePath()}/api/ml/jobs/jobs_summary`, {
     method: 'POST',
     credentials: 'same-origin',
-    body: JSON.stringify({ jobIds }),
+    body: JSON.stringify({}),
```

Comment:
Tested this and yes, that body is required to be on there and an empty object or the api will crash.

Thanks for putting this here, I would suggest maybe just a comment that this is intentful and not a mistake to require an empty body to be sent?

Reply:
Yeah, I tried removing the body when refactoring and came up against the `500` so it got reinstated by popular demand. I'll add a comment that clarifies this -- thanks for calling it out and good catch! 🙂